index.bs

```diff
@@ -266,12 +294,15 @@ The <dfn export>distance fraction</dfn> of a {{Document}} |D| is the lesser of
     distance</a>, or
 * 1.0.
 
-The <dfn export>impact region</dfn> of a {{Document}} |D| is the set containing
+The <dfn export>node impact region</dfn> of an <a>unstable</a> {{Node}} |N|
+is the set containing
```

Comment:
We can't define the region as a rectangle because it isn't rectangular.

This feels like a rehash of some discussions that occurred when I first wrote this spec.  Calling the impact region a set of points isn't new to this PR.

My view (then and now) is that a "set of points" is a familiar concept with a precise meaning from the field of mathematics, and the utility of the spec does not suffer by its employment.  (All specs necessarily use some foundational concepts without explicit definition - for example, CSS2 doesn't explain what a "percentage" is, and Paint Timing doesn't explain what a "pixel" is.)

So it's hard for me to see any issue here in the shoes of a browser engineer, the target audience of a web spec.  But if you have a different way of defining impact regions that you'd prefer to see I am open to considering it.

Reply:
Oh apologies if I've said this before... Ok, let's not block this PR on that.